docassemble/AssemblyLine/al_document.py

```diff
@@ -654,16 +658,24 @@ def init(self, *pargs, **kwargs):
         self.always_enabled = hasattr(self, "enabled") and self.enabled
 
     def as_pdf(
-        self, key: str = "final", refresh: bool = True, pdfa: bool = False
+        self,
+        key: str = "final",
+        refresh: bool = True,
+        pdfa: bool = False,
+        append_suffix_test: str = "preview",
```

Comment:
Yes, open to a new name for that parameter. I didn't want to call it "append_suffix" because I also add a `_` to the suffix, but maybe that's not crucial.